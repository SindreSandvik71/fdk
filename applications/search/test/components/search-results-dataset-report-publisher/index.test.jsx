import React from 'react';
import { shallow } from 'enzyme';
import SearchPublishers from '../../../src/components/search-results-dataset-report-publisher';

let onSearch;
let defaultProps;
let wrapper;

beforeEach(() => {
  onSearch = jest.fn();
  defaultProps = {
    onSearch,
    value: 'STAT'
  };
  wrapper = shallow(<SearchPublishers {...defaultProps} />);
});

test('should render SearchPublishers correctly with props', () => {
  expect(wrapper).toMatchSnapshot();
});
