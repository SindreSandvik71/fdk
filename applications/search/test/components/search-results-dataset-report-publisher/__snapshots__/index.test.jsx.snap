// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchPublishers correctly with props 1`] = `
<div
  className="section fdk-report-search-publishers"
>
  <div
    className="fdk-report-search-publishers__header mb-2"
  >
    Velg virksomhet
  </div>
  <Async
    autoload={true}
    backspaceRemoves={true}
    cache={Object {}}
    ignoreAccents={false}
    ignoreCase={true}
    labelKey="name"
    loadOptions={[Function]}
    loadingPlaceholder="Loading..."
    multi={false}
    options={Array []}
    placeholder="Søk på virksomhetsnavn..."
    searchPromptText="Skriv for å søke"
    value="STAT"
    valueKey="orgPath"
  />
</div>
`;
